.github/autobump.txt

```diff
@@ -1926,4 +1927,4 @@ zpaqfranz
 zsh-vi-mode
 zshdb
 zstd
-zydis
```

Comment:
What's happening here? Is it an "end of file newline" issue?

Reply:
yeah, I think so. Let me turn it back (I turned off while doing another PR)